protocols/kad/src/behaviour.rs

```diff
@@ -867,6 +904,11 @@ where
     ///
     /// > **Note**: Bootstrapping requires at least one node of the DHT to be known.
     /// > See [`Behaviour::add_address`].
+    /// > **Note**: Bootstrap does not require to be called manually. It is automatically
+    /// invoked at regular intervals based on the configured bootstrapping interval.
+    /// The bootstrapping interval is used to call bootstrap periodically
+    /// to ensure a healthy routing table.
+    /// > See bootstrap_interval field in Config.
```

Comment:
```suggestion
    /// > See [`Config::bootstrap_interval`] field in Config.
```

How about a link here?

Reply:
Fixed